Horion/DrawUtils.cpp

```diff
@@ -506,3 +506,25 @@ void DrawUtils::drawLinestrip3d(const std::vector<vec3_t>& points) {
 
 	tess_end(game3dContext, myTess, entityFlatStaticMaterial);
 }
+
+void DrawUtils::drawHologram(vec3_t pos, std::string text, float size) {
```

Comment:
Creating a new function in DrawUtils thats basically just `drawNameTags()` (which already was borderline) with changed colors is really not needed

Reply:
There isn't a way to draw a hologram/nametag without the entity... which is why I had to add `DrawUtils::drawHologram()`

And using black and 0.5 for transparency would look better for that purpose

Reply:
nice! any way to add like color to the waypoint or maybe edit it in a gui?